static/app/views/alerts/utils/getIncidentRuleDiscoverUrl.tsx

```diff
@@ -35,7 +36,7 @@ export function getIncidentRuleDiscoverUrl(opts: {
     name: (rule && rule.name) || '',
     orderby: `-${getAggregateAlias(rule.aggregate)}`,
     yAxis: rule.aggregate,
-    query: (rule?.query || eventType) ?? '',
+    query: eventTypeTagFilter ? eventTypeTagFilter : (rule?.query || eventType) ?? '',
```

Comment:
maybe we can just fall through them in order instead of ternary
```suggestion
    query: eventTypeTagFilter || rule?.query || eventType ?? '',
```